web/src/components/text-input/search-field.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Is it covered by `aria`, correct? We can disable it then I think in general config

Reply:
The label has an associated control but it doesn't validate it because of spreading the props and not explicit setting it